mne/viz/_3d.py

```diff
@@ -464,8 +516,39 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
                                'number of elements as PySurfer plots that '
                                'will be created (%s)' % n_split * n_views)
 
-    subjects_dir = get_subjects_dir(subjects_dir=subjects_dir)
+    # Set default transparency if user didn't specify it
+    if transparent is None:
+        if colormap == 'mne_analyze':
+            transparent = False
+        else:
+            transparent = True
+
+    ctrl_pts = [fmin, fmid, fmax]
 
+    # Check if using old fmin/fmid/fmax cmap behavior
+    if limits is None:
+        # Throw deprecation warning
+        warnings.warn('Using fmin, fmid, fmax is deprecated and will be'
+                      ' removed in v0.10. Use "limits" instead.',
+                      DeprecationWarning)
+        for fi, f in enumerate(ctrl_pts):
```

Comment:
I just need to fill in the flim value if they're using the old method and left any of the limits as None. It's a consequence of removing the defaults from the function header.